ql/src/java/org/apache/hadoop/hive/ql/ddl/table/partition/exchange/AlterTableExchangePartitionsDesc.java

```diff
@@ -53,4 +54,20 @@ public Table getDestinationTable() {
   public Map<String, String> getPartitionSpecs() {
     return partitionSpecs;
   }
+
+  @Override
+  public void setWriteId(long writeId) {
+    this.writeId = writeId;
+  }
+
+  @Override
+  public String getFullTableName() {
+    return null;
```

Comment:
intentional that this returns null?

Reply:
Yes

Reply:
Could we add a comment for this then, why it is needed? This seems unintuitive..

Reply:
Was not sure what to return for table name, so disabled advancing the write ID, as this DDL involves two different tables. I will just revert the entire file, till I figure this out.